.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,19 @@
+  name: Java-Selenium CI
+  on:
+    push:
+      branches: [main]
+    pull_request:
+      branches: [main]
+    workflow_dispatch:
+  jobs:
+    CI:
+      runs-on: windows-latest
+      steps:
+        - uses: actions/checkout@v2
+        - name: Set up JDK 11
+          uses: actions/setup-java@v2
+          with:
+            java-version: '11'
+            distribution: 'adopt'
+        - name: Build with Maven
+          run: mvn --batch-mode --update-snapshots verify
```

Comment:
verify execute: verify and package, these steps are not needed

Reply:
It was fixed